src/RecipeEntity.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace GRUB;
+namespace GRUB\recipe;
```

Comment:
```suggestion
namespace GRUB\Recipe;
```
should be capital R probably, I'm worried it'll throw an error if it's not exactly right